tests/testthat/test-namespace_linter.R

```diff
@@ -85,7 +85,7 @@ test_that("returns the correct linting", {
 
   expect_lint(
     "stats::sd(c(1,2,3))\nstats::sdd(c(1,2,3))",
-    list(line = "stats::sdd(c(1,2,3))"),
+    list(line = c("2" = "stats::sdd(c(1,2,3))")),
```

Comment:
PTAL -- consequence of shifting to `xml_nodes_to_lints()` -- unsure if this is fine or maybe reflects something to edit in `xml_nodes_to_lints()` -- `lines[[line1]]`? any reason not to strip attributes?

Reply:
ok. I'll do that in separate PR in case it breaks other stuff. as well as add some more direct tests of xml_nodes_,to_lints() which are clearly needed!